portafly/doc/ux_specs/global_components/alerts.md

```diff
@@ -0,0 +1,24 @@
+# Alerts UX/UI specs
+
+[Back to index](../index.md)
+
+* [PatternFly Alerts design guidelines](https://www.patternfly.org/v4/design-guidelines/usage-and-behavior/alerts-and-notifications)
+* [PatternFly-react Alerts docs](https://www.patternfly.org/v4/documentation/react/components/alert)
+
+### Network error alerts
+* Network errors are displayed in **toast alerts** ([PF specs](https://www.patternfly.org/v4/documentation/react/components/alert#types)), unless the current state of the UI is blocking users' view of the page layout (eg. modal dialogs)
+  * In this case, the errors will be displayed as **inline alerts** ([PF specs](https://www.patternfly.org/v4/documentation/react/components/alert#inline-types)), making sure the positioning is as close as possible to the trigger (eg. button) of the action that launched the network request.
+* Network error alerts are persistent and always display a dismiss button (`fa-times` icon)
+* Network error alerts variant is `warning`
+* Network error types:
```

Comment:
@josemigallas @damianpm could you please elaborate on that? Or shall we better spec out the type of error in the page or component file where the alert is displayed?

Reply:
A network error related to the connection, not the server. They can be:

- Secure connection failed
- Timed out
- Cannot find host
- Not connected to Internet
- Cannot connect to host

On the other hand, server errors include:

- Not found
- Authentication error
- Server internal error
etc

Reply:
@josemigallas cool, thank you! I will update the spec.
about server errors, aren't those usually displayed in full pages (thinking of 404 pages for instance). Could they be triggered by bulk actions as well?

Reply:
Yes, server errors are displayed in it's own page, for example a 404 needs its own full page because there is no other page where you can insert an alert.